action.yml

```diff
@@ -41,3 +45,4 @@ runs:
         NOMAD_JOB: ${{ inputs.job }}
         NOMAD_NAMESPACE: ${{ inputs.namespace }}
         PARAMETERIZED_JOB: ${{ inputs.parameterized }}
+        ALLOW_FAILURE: ${{ inputs.allow-failure}}
```

Comment:
Same, here, the ENV var name is not that good here.

Reply:
renamed to allow-stderr